### PR TITLE
kata-deploy: fix k8s 1.13 example files

### PR DESCRIPTION
**Pull request by egernst**
Fixes: #560

Signed-off-by: Eric Ernst <eric.ernst@intel.com>

### CODE REVIEW

File: kata-deploy/k8s-1.13/kata-qemu-runtimeClass.yaml [hunk 1]
```diff
@@ -2,11 +2,13 @@
 kind: RuntimeClass
 apiVersion: node.k8s.io/v1alpha1
 metadata:
-    name: kata
-handler: kata
+    name: kata-qemu
+spec:
+    runtimeHandler: kata-qemu
```
Comment (grahamwhaley) [inline kata-deploy/k8s-1.13/kata-qemu-runtimeClass.yaml]: I'm presuming the order swap of `kata` and `kata-qemu` here is benign?

Comment (egernst) [reply kata-deploy/k8s-1.13/kata-qemu-runtimeClass.yaml]: yes

**Comment by egernst** [comment]
I updated the README.md to describe the need to apply the CRD when using K8S < 1.14.  Also s/k8s/Kubernetes in the text.

File: kata-deploy/k8s-1.13/runtimeclass-crd.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    kind:     RuntimeClass
+  scope: Cluster
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            runtimeHandler:
+              type: string
+              pattern: '^([a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*)?$'
```
Comment (jodh-intel) [inline kata-deploy/k8s-1.13/runtimeclass-crd.yaml]: Can a handler *really* start with a number?

Comment (egernst) [reply kata-deploy/k8s-1.13/runtimeclass-crd.yaml]: apparently.  This was pulled from K8S - I don't maintain the regex..

Comment (jodh-intel) [reply kata-deploy/k8s-1.13/runtimeclass-crd.yaml]: Seems like it might be a bug, but let's be consistent with k8s I guess... ;)

**Pull request merged**